client-core/src/synchronizer/auto_sync_data.rs

```diff
@@ -20,6 +53,16 @@ pub struct AutoSyncData {
     pub max_height: u64,
     /// current syncing wallet name
     pub wallet: String,
+    /// all wallets
+    pub wallets_all: Vec<String>,
```

Comment:
I suggest to change to `syncing_wallet/current_wallet` and `unlocked_wallets`.

Reply:
ok, i'll change to 
current_wallet,
unlocked_wallets